meta-resin-common/recipes-support/resin-device-progress/resin-device-progress/resin-device-progress

```diff
@@ -67,9 +67,9 @@ if [ `jq ".deviceId | length" $CONFIG_PATH` -eq 0 ]; then
 fi
 
 # If the user api key exists we use it instead of the deviceApiKey as it means we haven't done the key exchange yet
-read device_api_key device_id <<<$(jq -r '.apiKey // .deviceApiKey,.deviceId' $CONFIG_PATH)
+_device_api_key=${PROVISIONING_API_KEY:-DEVICE_API_KEY}
```

Comment:
Nope, this is for the case where an old device has had a host update and hasn't yet done a key exchange so it only has a "provisioning key" (which is really a provisioning+device key) eg if supervisor hasn't been updated to one that supports key exchange or hasn't had a chance to do it yet